dashboard/test/ui/features/acquisition_products/pegasus_landing_pages_2.feature

```diff
@@ -15,5 +15,3 @@ Examples:
   | http://code.org/athletes                                          | athletes tutorial landing  |
   | http://code.org/tools/applab                                      | app lab tutorial landing   |
   | http://code.org/dance                                             | dance tutorial landing     |
-  | http://code.org/educate/resources/videos                          | video resources landing    |
-#  | http://code.org/educate/resources/videos?force_youtube_fallback=1 | resources yt fallback      |
```

Comment:
Can we talk out loud in sprint planning about removing these?

Reply:
Yes of course! I also know we'll need to address the YouTube header issue as well.

Reply:
After what we talked about in sprint planning, I'm going to merge this PR as-is and prioritize working with Kelby on the YT titles and adding ignore regions as @dmcavoy requested to maintain maximum coverage on other pages.